modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/ObjectStatisticsImpl.java

```diff
@@ -24,7 +24,7 @@
  */
 public class ObjectStatisticsImpl implements Cloneable, ObjectStatistics {
     /** Total number of rows in object. */
-    private final long rowsCnt;
```

Comment:
I would prefer to have it final, cause it prevent from having issues in case of multithreaded modification

Reply:
It used in the merge process where we can clone an initial object (to save all previously collected statistics) and set newly collected data. I'll think about how to fix it.